demos/src/demo-grouped-bar.js

```diff
@@ -1,22 +1,20 @@
-'use strict';
+import { select, selectAll } from 'd3-selection';
+import { default as PubSub } from 'pubsub-js';
```

Comment:
this is odd,  es6 by default will export on 'default ' when you do export default. 

meaning, `import PubSub from 'pubsub-js'` should have worked.

Reply:
True, changing. Thanks!